packages/gatsby-source-wikipedia/src/fetch.js

```diff
@@ -70,16 +70,16 @@ const search = ({ query, limit }) =>
         return {
           title,
           description: descriptions[i],
-          id: /en.wikipedia.org\/wiki\/(.+)$/.exec(urls[i])[1],
+          id: term,
         }
       })
     )
 
-const getArticle = name =>
-  fetch(`https://en.m.wikipedia.org/wiki/${name}?action=render`)
+const getArticle = name => lang =>
+  fetch(`https://${lang}.m.wikipedia.org/wiki/${name}?action=render`)
     .then(res => res.text())
     .then(pageContent =>
-      pageContent.replace(/\/\/en\.wikipedia\.org\/wiki\//g, `/wiki/`)
+      pageContent.replace(`/\/\/${lang}\.wikipedia\.org\/wiki\//g`, `/wiki/`)
```

Comment:
I don't think this will do what you expect. If we don't care about the language value in replacement, then this should be good:
```suggestion
      pageContent.replace(/\/\/[a-z]+\.wikipedia\.org\/wiki\//g, `/wiki/`)
```

But that also means that links in different languages will point to the same URL. This works when you deal with a single language. But won't work that well for multilanguage setups.

I assume a multilanguage setup is a rare case and we can address it separately if someone actually needs it.